integrations/lambda/src/lambda_sample/lambda_helper.py

```diff
@@ -0,0 +1,296 @@
+from boto3 import session
+from botocore.auth import SigV4Auth
+from botocore.awsrequest import AWSRequest
+from http import HTTPStatus
+from requests.exceptions import HTTPError
+import json
+import os
+import requests
+import time
+import zipfile
+
+MAX_WAIT_SECONDS = 900 # 15 minutes
+
+def create_lambda(session: session, lambda_name: str, database_name: str, table_name: str, role_name: str,
+                  mem_store_retention_period_in_hours=12, mag_store_retention_period_in_days=3653, batch_size=100) -> str:
```

Comment:
can we align these defaults with TImestream console's "Create Table"'s defaults?

Reply:
The default values for `mem_store_retention_period_in_hours` and `mag_store_retention_period_in_days` match the default values in the Timestream console, 12 hours and 10 years respectively.